ops/Makefile

```diff
@@ -1,3 +1,12 @@
+build:
+	DOCKER_BUILDKIT=1 \
+	docker-compose \
+		-f docker-compose.yml build builder
+	DOCKER_BUILDKIT=1 \
+	docker-compose \
+		-f docker-compose.yml build
+.PHONY: build
+
 up: down
 	DOCKER_BUILDKIT=1 \
 	docker-compose \
```

Comment:
Should the `up` command also be modified so that it runs `dc build builder` before it runs `dc up --build`?